package-lock.json

```
@@ -1,3 +1,3948 @@
 {
-  "lockfileVersion": 1
+  "requires": true,
+  "lockfileVersion": 1,
+  "dependencies": {
+    "@types/node": {
+      "version": "10.12.18",
+      "resolved": "https://registry.npmjs.org/@types/node/-/node-10.12.18.tgz",
+      "integrity": "sha512-fh+pAqt4xRzPfqA6eh3Z2y6fyZavRIumvjhaCL753+TVkGKGhpPeyrJG2JftD0T9q4GF00KjefsQ+PQNDdWQaQ==",
+      "dev": true
+    },
+    "@types/semver": {
+      "version": "5.5.0",
+      "resolved": "https://registry.npmjs.org/@types/semver/-/semver-5.5.0.tgz",
+      "integrity": "sha512-41qEJgBH/TWgo5NFSvBCJ1qkoi3Q6ONSF2avrHq1LVEZfYpdHmj0y9SuTK+u9ZhG1sYQKBL1AWXKyLWP4RaUoQ==",
+      "dev": true
+    },
+    "PrettyCSS": {
+      "version": "0.3.17",
+      "resolved": "https://registry.npmjs.org/PrettyCSS/-/PrettyCSS-0.3.17.tgz",
+      "integrity": "sha512-E6RUMvuq41DqA9uvgqrinYrug3SpgamH1HEeF9zy9HHYxwoAW6fBKmf/NDoNl0MWspLReaKXFNhjTYwZy3bmlQ==",
+      "dev": true,
+      "requires": {
+        "option-parser": "~1.0.0"
```

```suggestion
        "option-parser": "1.0.0"
```



<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>